scripts/installer/glassfish-setup.sh

```diff
@@ -65,7 +74,7 @@ function preliminary_setup()
   ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.host=${RSERVE_HOST}"
   ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.port=${RSERVE_PORT}"
   ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.user=${RSERVE_USER}"
-  ./asadmin $ASADMIN_OPTS create-jvm-options "\-Ddataverse.rserve.password=${RSERVE_PASS}"
+  ./asadmin $ASADMIN_OPTS create-jvm-options '\-Ddataverse.rserve.password=${ALIAS=reserve_password_alias}'
```

Comment:
Beware: there is a typo in `reserve_password_alias` - this should be `rserve_password_alias`.

Reply:
@poikilotherm good catch! I fixed the typo in 5de567d. I have not tried this code myself.

Reply:
I have tried it, it works.